src/neml2/models/solid_mechanics/PerzynaPlasticFlowRate.cxx

```diff
@@ -39,8 +39,9 @@ PerzynaPlasticFlowRate::expected_params()
 
 PerzynaPlasticFlowRate::PerzynaPlasticFlowRate(const ParameterSet & params)
   : PlasticFlowRate(params),
-    _eta(register_parameter("eta", Scalar(params.get<Real>("reference_stress")), false)),
-    _n(register_parameter("n", Scalar(params.get<Real>("exponent")), false))
+    _eta(register_parameter(
+        "eta", Scalar(params.get<Real>("reference_stress"), default_tensor_options), false)),
+    _n(register_parameter("n", Scalar(params.get<Real>("exponent"), default_tensor_options), false))
```

Comment:
Can we have a shortcut method for `register_parameter("whatever", Scalar(...), default_tensor_options), false)`?  Like `register_scalar_parameter`?  We use this all over...

Reply:
Yes, let me add that...